src/analysis/tunnel_graph.py

```diff
@@ -271,6 +271,17 @@ def plot_throughput_graph(self):
 
             if flow_id in self.ingress_tput and flow_id in self.ingress_t:
                 empty_graph = False
+                
+                #output data to csv file
+                x_values = self.ingress_t[flow_id]
+                y_values = self.ingress_tput[flow_id]
+
+                with open('./LSTM/data.csv', 'w') as f:
+                    fields = ["Time","Throughput"]
+                    writer = csv.writer(f)
+                    writer.writerow(fields)
+                    writer.writerows(zip(x_values,y_values))
+
```

Comment:
intersting. is there a reason why ingress data are recorded for analysis but not egress

Reply:
@andrewberryy